tutor/test_t1_37_AccountMaintenance.py

```diff
@@ -412,10 +417,14 @@ def test_admin_impersonate_a_user_8253(self):
         self.admin.driver.find_element(By.ID, 'query').send_keys('student01')
         self.admin.driver.find_element(
             By.XPATH, '//input[@value="Search"]').click()
-        self.admin.wait.until(
+        element = self.admin.wait.until(
             expect.element_to_be_clickable(
-                (By.XPATH, '//a[contains(text(),"Sign in as")]')
+                (By.XPATH, '//td[text()="Atticus Finch"]')
             )
```

Comment:
Can you rework this so the username in the XPath is obtained by a variable as opposed to hard coding it?